ecommers/lib/core/common/mapper.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:ecommers/core/models/index.dart';
+import 'package:ecommers/ui/decorations/index.dart';
+
+class Mapper {
```

Comment:
do we need this mapper? Consider creating factory constructor at OrderModel

Reply:
fixed